src/panel/app_panel.js

```diff
@@ -4,7 +4,7 @@ import PanelsView from '../views/app_panel.dot';
 import Panel from '../libs/panel';
 import FavoritePanel from './favorites_panel';
 import PoiPanel from './poi_panel';
-import ServicePanel from './service_panel';
+import ServicePanel from './servicePanel';
```

Comment:
Just a naming convention of React. It's better to use capitalized named for component files. It's a consistency reason, because when used in JSX, custom components *must* start with a capital (lowercase is reserved for HTML tags).